charts/plex-media-server/templates/statefulset.yaml

```diff
@@ -0,0 +1,192 @@
+      {{- if and .Values.rclone.enabled .Values.rclone.configSecret }}
+      {{- range .Values.rclone.remotes }}
+      - name: rclone-media-{{ (split ":" .)._0 }}
+        emptyDir: {}
+      {{- end }}
+      - name: rclone-config
+        emptyDir: {}
+      - name: rclone-config-data
+        secret:
+          secretName: {{ .Values.rclone.configSecret }}
+      {{- end }}
+      {{- if .Values.initContainer.script }}
+      - name: init-script-configmap
+        configMap:
+          defaultMode: 0700
+          name: {{ include "pms-chart.fullname" . }}-init-script
+      {{- end }}
+      {{- if .Values.extraVolumes }}
+{{ toYaml .Values.extraVolumes | indent 6 }}
+      {{- end }}
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      terminationGracePeriodSeconds: 120
+      initContainers:
+      {{- if .Values.initContainer.script }}
+      - name: {{ include "pms-chart.fullname" . }}-pms-init
+        image: {{ include "pms-chart.init_image" . }}
+        command: ["/init/init.sh"]
+        volumeMounts:
+        - name: pms-config
+          mountPath: /config
+        - name: init-script-configmap
+          mountPath: /init
+      {{- if .Values.extraVolumeMounts }}
+{{ toYaml .Values.extraVolumeMounts | indent 8}}
+      {{- end }}
+      {{- end }}
+      {{- if and .Values.rclone.enabled .Values.rclone.configSecret }}
+      - name: {{ include "pms-chart.fullname" . }}-config
+        image: {{ include "pms-chart.init_image" . }}
+        command:
+          - cp
+        args:
+          - -v
+          - /in/rclone.conf
+          - /out/rclone.conf
+        volumeMounts:
+        - name: rclone-config-data
+          mountPath: /in
+          readOnly: true
+        - name: rclone-config
+          mountPath: /out
+      {{- end }}
+      containers:
+      - name: {{ include "pms-chart.fullname" . }}-pms
+        image: {{ include "pms-chart.image" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+        - containerPort: 32400
+          name: pms
+        env:
+        - name: PLEX_CLAIM
+          value: {{ .Values.pms.claim }}
+        - name: HOSTNAME
+          value: {{ .Values.pms.name}}
+        - name: TZ
+          value: {{ .Values.pms.timeZone }}
+        {{- if .Values.pms.updateChannel }}
+        - name: PLEX_UPDATE_CHANNEL
+          value: {{ .Values.pms.updateChannel }}
+        {{- end }}
+        {{- if .Values.pms.uid }}
+        - name: PLEX_UID
+          value: {{ .Values.pms.uid }}
+        {{- end }}
+        {{- if .Values.pms.gid }}
+        - name: PLEX_GID
+          value: {{ .Values.pms.gid }}
+        {{- end }}
+        {{- if .Values.pms.allowedNetworks }}
+        - name: ALLOWED_NETWORKS
+          value: {{ .Values.pms.allowedNetworks }}
+        {{- end }}
+        {{- if .Values.pms.advertiseIp }}
+        - name: ADVERTISE_IP
+          value: {{ .Values.pms.advertiseIp }}
```

Comment:
`ADVERTISE_IP` should match the ingress hostname, single value for both entries.